WheelOfFortune/Game.cs

```diff
@@ -91,7 +96,60 @@ public void StartRound()
         /// </summary>
         public void StartTurn()
         {
-            Console.WriteLine(CurrentPuzzle.PuzzleSoFar + "\n");
+            // set new CurrentPlayer if CurrentPlayer is null
+            if(CurrentPlayer == null)
+            {
+                // dequeue from Players list and set to CurrentPlayer aka take next player from queue and set it to current
+                CurrentPlayer = Players.Dequeue();
+            } 
+
+            // so now that we have a current player, they will start to perform guesses so call on StartTurn method from Prompt class passing in CurrentPlayer and CurrentPuzzle
+            Prompt.StartTurn(CurrentPlayer, CurrentPuzzle);
+
+            // the GetAction method from Prompt class should return 1 or 2 from the Enum 
+            char solveOrGuessActionType = Prompt.GetActionType();
+
+            // Take return value from Prompt.GetActionType method and pass in action type to CurrentPlayer.SetAction
+            // This will not return anything, it just sets the action in CurrentAction 
+            CurrentPlayer.SetAction(solveOrGuessActionType);
```

Comment:
So for this function, the expected input is of type `Action.ActionType` which is our enum in action. The different ways you can access this type is either by using `Action.ActionType.SolvePuzzleAction`/`Action.ActionType.GuessLetterAction` OR you can access them with numbers (think kind of like indexes in js) in this kind of format `(Action.ActionType)0`/`(Action.ActionType)1`. This may require us to either typecast the response or update the GetActionType to return an int value

Reply:
Ah gotcha! I remember seeing this similarly in the Prompt class for the PromptAction method. I wasn't sure if it was this or if I should be using the return value from Prompt.GetActionType which is why I passed in the stored variable. I believe it is currently returning a string so I can cast it to return an int instead.